### DISCUSSION TITLE
tokio v1.17.0 upgrading

### DISCUSSION THREAD

**Discussion by knight110001** [HarukaMa/aleo-prover#1] [Q&A]
@HarukaMa Did you ever upgrade tokio version from 1.15.0 to 1.17.0? After upgrade to v1.17.0, it seems something blocking the task channel after several rounds done.

**Comment by HarukaMa** [comment]
I’ve resolved a blocking issue in 0.3.0 beta, is it still happening in the latest version?

**Comment by knight110001** [comment]
Nice, I will try again, thanks!

**Comment by HarukaMa** [comment]
Maybe off topic, but I see that in your rayon test code, the owner of the coinbase in the hardcoded block template is `...5kycny`. Do you own that address?

**Comment by knight110001** [comment]
Yes, I have tested the latest version, it seems the issue has gone. some guy's friend owns that address.
